tests/Custom/DesiredCapabilitiesTest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Behat\Mink\Tests\Driver\Custom;
+
+use Behat\Mink\Driver\Selenium2Driver;
+use Behat\Mink\Tests\Driver\TestCase;
+
+class desiredCapabilitieTest extends TestCase
+{
+
+    public function testGetDesiredCapabilties()
+    {
+        $driver = $this->getSession()->getDriver();
+        $this->assertNotEmpty($driver->getDesiredCapabilities(), 'desiredCapabilities empty');
```

Comment:
I'm not sure if we need to assert content of desired capabilities at all. I thought about asserting:

1. that we have some capabilities at the start
2. what is set we're getting back
3. if we set during active session, then we get proper exception (exception class & message)